### ISSUE TITLE
Cannot access website from Russia. Russia is blacklisted.

### ISSUE THREAD

**Issue by PixelSymbols** [swagger-api/swagger-ui#9640]
OS: windows
Browser: chrome
country: Russia
Version: 23
Method: opening website
swagger-ui: any
swagger: any

### Content & configuration

[Website](https://swagger.io/) blocks people from Russia on accessing it.

### Screenshots
![image](https://github.com/swagger-api/swagger-ui/assets/81364140/f5fbe916-2b92-49c5-bb98-7de201d02727)

### How can we help?
Remove Russia from blacklist

**Comment by aliakseis** [comment]
Wer mit Ungeheuern kämpft, mag zusehn, dass er nicht dabei zum Ungeheuer wird. Und wenn du lange in einen Abgrund blacklist, blickt der Abgrund auch in dich hinein.

**Comment by satokenta940** [comment]
This is really unfortunate, I remember [Apidog ](https://apidog.com/)Russia is accessible, you can give it a try, this is also an API documentation tool.

**Comment by pv4** [comment]
I have the same issue.

Any progress on this? This doesn't look like a hard-to-fix issue.

**Comment by aliakseis** [comment]
### **Ich bin Schwager!!!**

https://be.wikipedia.org/wiki/%D0%A8%D0%B2%D0%B0%D0%B3%D0%B5%D1%80

**Comment by PixelSymbols** [comment]
Well, 2 months passed. We can confirm, swagger are racists.

**Comment by aliakseis** [comment]
On https://swagger.io we see: "API Development for Everyone", "SmartBear is committed to Open Source development" etc.

On https://smartbear.com/terms-of-use/: 
"Customer shall not sublicense, distribute, or otherwise make available SmartBear: (a) in any Sanctioned Jurisdiction or to a Sanctioned Person"

Open Source licenses are designed to ensure that software can be freely used, modified, and shared. According to the Open Source Initiative (OSI), which maintains the Open Source Definition, and the Free Software Foundation (FSF), which defines Free Software, an Open Source license must not discriminate against any person or group of persons.

Therefore, a license that restricts the use of the software in a "Sanctioned Jurisdiction" would not meet the OSI's criteria for being considered Open Source. The same applies to the FSF's definition of Free Software. For example, the GPL family of licenses, which includes popular licenses like GPL, GPLv2, and GPLv3, specifically prevent the addition of further restrictions such as prohibiting the use of the software in certain countries.

The OSI and FSF emphasize that Open Source and Free Software licenses must allow anyone to use the software for any purpose, without discrimination against fields of endeavor or against persons or groups. This means that adding a clause to an Open Source license that prohibits usage in a sanctioned jurisdiction would violate these principles.

Modern examples of Virtue Signaling are posting stuff that you do not share in order to gain popularity and reputation. Another could be jumping on a trend in order to look better to your peers, no matter what your opinion on the trend is.

**Comment by falur** [comment]
Да, больше того они теперь если ты вошёл с ip российского а потом vpn ещё и блочат vpn ip

**Comment by BeMySlaveDarlin** [comment]
> Да, больше того они теперь если ты вошёл с ip российского а потом vpn ещё и блочат vpn ip

При переключении на не РФ айпишники без проблем открывается. Вероятно просто ваш VPN сервис в блеклисте независимо от настроек cloudflare

**Comment by cybergazer** [comment]
It's not just swagger, America is an incredibly racist and evergrowing totalitarian country which is about to roll out a policy to block Russians from contributing to GPL'd software, solely because of ethnicity. It's not they're blocking specific people who started the war in ukraine, no, anyone who's russian gets blocked. Just like all the russian maintainers who were banned from contributing to the linux kernel. This is america and how "free" and "democratic" it actually is.

Sources are here:
https://www.presidency.ucsb.edu/documents/executive-order-14071-prohibiting-new-investment-and-certain-services-the-russian

https://www.youtube.com/watch?v=L5Ec5jrpLVk

**Comment by JaneGame** [comment]
Кто-то может подсказать какой впн можно использовать, чтобы работать со сваггером?

**Comment by PixelSymbols** [comment]
> Кто-то может подсказать какой впн можно использовать, чтобы работать со сваггером?

и у тебя после их расисткого поведения осталось желание использовать их продукт? Просто используй альтернативы

**Comment by JaneGame** [comment]
К сожалению, эта штука ещё много где используется и мне необходим доступ к документации=( Другое дело, что старые ресурсы ещё работают, а доступ к самому сайту закрыт.

**Comment by aliakseis** [comment]
https://www.tiktok.com/@rock_pop80/video/7436058771991874848

**Comment by Gurgen-jpg** [comment]
Cancel Swagger

**Comment by Artmen1989** [comment]
Насамом деле - это действительно "позорно" закрывать откртый код. {да пишу с ошибками} =) .. Есть какое-то кино про это, там где разрабы специально делали так, что бы исходный код был "достоянием для всех", без предрасудков. И тут спусят время, когда я посмотрел это кино.. Тут такая ситуация, что всё закрывают. Это бред. И действительно нарушает "принципы открытости". Получется, что все компании лгут, что у них всё открыто для всех. До тех пор, пока им не будет выгодно "закрыть" это. 
И о какой "свободе, открытости - мы говорим?".. Позор.. и мне очень грустно от этого, что люди так делают... снова и снова. И я уверен, что "те люди" которых уже нет - и которые первоначально внесли огромный вклад в это. Не будут рады этому. Ни в прошлом, ни в настоящем и будущем.  (с) Мы закрыли Оpen-Source решение - по причине санкций и глобальной политики.. Так как мы "знаем" что мы не имеем "право" закрывать это! Тем не менее, мы это сделали. Потому, что мы боимся. Кого или чего? МИР - не рухнет. Просто люди, сами виноваты в этом. А не код. Люди - которые "вносят вклад" в развитие решений - спасибо ВАМ!! (/◕ヮ◕)/ <3
